app/src/main/java/com/dessalines/thumbkey/keyboards/DEMessageEaseSymbols.kt

```diff
@@ -914,18 +1102,416 @@ val KB_DE_MESSAGEEASE_SYMBOLS_SHIFTED =
             ),
             listOf(
                 SPACEBAR_KEY_ITEM,
-                RETURN_KEY_ITEM,
+                KB_DE_MESSAGEEASE_SYMBOLS_RETURN_KEY_ITEM,
+            ),
+        ),
+    )
+
+val KB_DE_MESSAGEEASE_SYMBOLS_NUMERIC_KEYBOARD =
```

Comment:
Get rid of this, we have a common numeric messageease keyboard now in `app/src/main/java/com/dessalines/thumbkey/keyboards/NumericMessagEase.kt`

Reply:
No probs. Apparently messageease has custom numeric keyboards for each language, but they should live in their own files, IE like `app/src/main/java/com/dessalines/thumbkey/keyboards/ENMessagEaseNumeric.kt`